<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rigging Toolkit</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Locators</string>
    </attribute>
    <widget class="QPushButton" name="L_BuildLocators">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Build Locators</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="rootHeightLabel">
        <property name="text">
         <string>Root Height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rootHeightLineEdit">
        <property name="text">
         <string>1.5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="spineCountLabel">
        <property name="text">
         <string>Spine Count</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="spineCountLineEdit">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="spineOffsetLabel">
        <property name="text">
         <string>Spine Offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="spineOffsetLineEdit">
        <property name="text">
         <string>0.25</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="spineBaseHeightLabel">
        <property name="text">
         <string>Spine Base Height</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="spineBaseHeightLineEdit">
        <property name="text">
         <string>1.75</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="fingersCountLabel">
        <property name="text">
         <string>Fingers Count</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="fingersCountLineEdit">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="phalanxCountLabel">
        <property name="text">
         <string>Phalanx Count</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="phalanxCountLineEdit">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="phalanxOffsetLabel">
        <property name="text">
         <string>PhalanxOffset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="phalanxOffsetLineEdit">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="legsOffsetLabel">
        <property name="text">
         <string>Legs Offset</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="legsOffsetLineEdit">
        <property name="text">
         <string>0.2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="scaleLabel">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="scaleLineEdit">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="L_MirrorLocators">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mirror locators</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Joints</string>
    </attribute>
    <widget class="QPushButton" name="J_GenerateJoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Joints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J_OrientJoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Orient Joints</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>90</y>
       <width>151</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="rigNameLabel">
        <property name="text">
         <string>Rig Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rigNameLineEdit">
        <property name="text">
         <string>BaseJointRig</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rigRootNameLabel">
        <property name="text">
         <string>RigRootName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rigRootNameLineEdit">
        <property name="text">
         <string>Rig_Root</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="zd">
    <attribute name="title">
     <string>IK</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ikRPsolver </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ikSCsolver</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ikSplineSolver</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>160</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="IK_FromJoint">
        <property name="text">
         <string>From Joint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="FromJointlineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="IK_ToJoint">
        <property name="text">
         <string>To Joint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ToJointlineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Handle</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="IK_BuildHandles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Build Handles</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Skin</string>
    </attribute>
    <widget class="QPushButton" name="S_BinSkinButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bind Skin</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="dropOffRateLabel">
        <property name="text">
         <string>Drop Off Rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dropOffRateLineEdit">
        <property name="text">
         <string>0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="skinMethodLabel">
        <property name="text">
         <string>Skin Method</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="skinMethodBox">
        <item>
         <property name="text">
          <string>Classical linear skinning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dual quaternion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weighted blend</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Controllers</string>
    </attribute>
    <widget class="QPushButton" name="C_BuildControllers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Build Controllers</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="locatorsGroupNameLabel">
        <property name="text">
         <string>Locators Group Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="locatorsGroupNameLineEdit">
        <property name="text">
         <string>Locators</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="G_ImportButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="G_ExportButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
